src/ui/zcl_abapgit_gui_router.clas.abap

```diff
@@ -598,8 +598,7 @@ CLASS ZCL_ABAPGIT_GUI_ROUTER IMPLEMENTATION.
     CASE is_event_data-action.
         " ZIP services actions
       WHEN zif_abapgit_definitions=>c_action-zip_import.                      " Import repo from ZIP
-        zcl_abapgit_zip=>import( lv_key ).
-        zcl_abapgit_services_repo=>refresh( lv_key ).
+        zcl_abapgit_services_repo=>import_zip( lv_key ).
```

Comment:
this reordering and moving `refresh` can be done as a separate PR

Reply:
not really, as PR removes `zip` methods so I can unroll `zcl_abapgit_services_repo=>import_zip` code here of course, but does it worth it ?